.gitignore

```diff
@@ -23,3 +23,7 @@ stdcerr
 # Python
 /.venv
 /.env
+
+# Conan
+conaninfo.txt
```

Comment:
I'm curious what kind of use case caused this file to be placed in the `libs/gil`?

I'm not an advanced user of Conan, but https://docs.conan.io/en/latest/reference/commands/consumer/info.html says

> -if INSTALL_FOLDER, --install-folder INSTALL_FOLDER
>                         local folder containing the conaninfo.txt and
>                         conanbuildinfo.txt files (from a previous conan
>                         install execution)

Reply:
I believe it is caused by me trying to manually try conan install, which makes the change erroneous

Reply:
let me try to clean clone it and run build again. I will report back about any changes

Reply:
Don't worry about that. It does not hurt to ignore it. Leave it as is.

You may notice, that despite of sharing several comments, I actually approved your PR without any request for changes. If I expected you to change something, you'd see my review status as not approved, but "changes requested" or something like that. You can find more about the review process on GitHub here https://help.github.com/en/articles/reviewing-proposed-changes-in-a-pull-request

Reply:
I'm resolving this conversation :-)